test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/MiscCryptoSuite.java

```diff
@@ -67,6 +70,42 @@ public static void main(String... args) {
 		);
 	}
 
+	private HapiApiSpec sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign(){
+		String sysAccount = "0.0.977";
+		String randomAccountA = "randomAccountA";
+		String randomAccountB = "randomAccountB";
+		String firstKey = "firstKey";
+		String secondKey = "secondKey";
+
+		return defaultHapiSpec("sysAccountKeyUpdateBySpecialWontNeedNewKeyTxnSign")
+				.given(
+						withOpContext((spec, opLog) -> {
+							spec.registry().saveKey(sysAccount, spec.registry().getKey(GENESIS));
+						}),
+						newKeyNamed(firstKey).shape(SIMPLE),
+						newKeyNamed(secondKey).shape(SIMPLE)
+				)
+				.when(
+						cryptoCreate(randomAccountA)
+								.key(firstKey),
+						cryptoCreate(randomAccountB)
+								.key(firstKey)
+								.balance(ONE_HUNDRED_HBARS)
+				)
+				.then(
+						cryptoUpdate(sysAccount)
+								.key(secondKey)
+								.payingWith(GENESIS)
+								.hasKnownStatus(SUCCESS)
+								.logged(),
+						cryptoUpdate(randomAccountA)
+								.key(secondKey)
+								.signedBy(firstKey)
+								.payingWith(randomAccountB)
```

Comment:
I would replace `randomAccountB` here with `GENESIS`, to emphasize that the network **only** waives the signing requirement when `GENESIS` is updating a system account---not a normal account.

Reply:
Does another system account (50) also have this privilege?